backend/src/services/preferences/wishlist.service.ts

```diff
@@ -66,6 +66,25 @@ const deleteWishlist = async (input: WishlistInput): Promise<void> => {
   return;
 };
 
+const getWishlistLike = async (input: WishlistInput): Promise<boolean> => {
+  const { userId, modelId, complectationId } = input;
+  const wishlist = await prisma.user_Wishlist.findFirst({
+    where: {
+      user_id: userId,
+      OR: [
+        {
+          model_id: modelId,
+        },
+        {
+          complectation_id: complectationId,
+        },
+      ],
+    },
+  });
```

Comment:
We do the same query on line 15 and in other methods. Could you please extract them into a reusable function?